packages/fxa-auth-server/lib/senders/emails/templates/fraudulentAccountDeletion/en.ftl

```diff
@@ -0,0 +1,8 @@
+fraudulentAccountDeletion-subject = Your { -brand-firefox } account was deleted
+fraudulentAccountDeletion-title = Your account was deleted
+fraudulentAccountDeletion-content = Recently, a { -brand-firefox } account was created and a subscription was charged using this email address. As we do with all new accounts, we asked that you confirm your account by first validating this email address.
+fraudulentAccountDeletion-content-2 = At present, we see that the account was never confirmed. Since this step was not completed, we are not sure if this was an authorized subscription. As a result, the Firefox account registered to this email address was deleted and your subscription was canceled with all charges reimbursed.
```

Comment:
```suggestion
fraudulentAccountDeletion-content-2 = At present, we see that the account was never confirmed. Since this step was not completed, we are not sure if this was an authorized subscription. As a result, the { -brand-firefox } account registered to this email address was deleted and your subscription was canceled with all charges reimbursed.
```